docs/index.rst

```diff
@@ -40,6 +39,16 @@ encoding
   Encoding to be used to read an OpenAPI spec. If not passed, Sphinx's
   source encoding will be used.
 
+paths
+  A comma separated list of paths to filter the included openapi spec by. For example:
+
+  .. code:: restructuredtext
+
+      .. openapi:: specs/openapi.yml
+          :paths: /persons,/evidence
```

Comment:
I did not switch the option name from `paths` to `endpoints`, as the more I think about it the more I think it makes sense to remain consistent with the naming conventions of the OpenAPI spec. It looks like I use `endpoints` in the docs in this string:
```
 Would only render the endpoints at ``/persons`` and ``/evidence``, ignoring all others.
```

Is that the reference you were referring to in your comment above? If so would it suffice to change the text there to `paths` instead?